testsuite/MDAnalysisTests/plugins/open_files.py

```diff
@@ -86,24 +86,25 @@ def formatFailure(self, test, err):
     def report(self, stream):
         """Display the number of open files at the end of the test suite.
         """
-        open_files = self._gather_open_files()
+        open_files = _gather_open_files()
         handle_couter = collections.Counter(open_files)
         print('\n', file=stream)
         print('By the end of the tests, there are {} open handle for {} files:'
               .format(len(open_files), len(handle_couter)), file=stream)
         for path, count in handle_couter.items():
             print('* ({}) {}'.format(count, path), file=stream)
 
-    def _gather_open_files(self):
-        """Return a list of the path to open files.
 
-        If several file handle are open for a given file, then the path will
-        appear as many time as they are open file handle for that file.
-        """
-        process = psutil.Process(os.getpid())
-        handlers = process.open_files()
-        path = [f.path for f in handlers]
-        return path
+def _gather_open_files(self):
```

Comment:
How can you be that fast!? The extra `self` stayed there for less than 5 
minutes!!!

On 13/06/16 09:45, kain88-de wrote:
>
> In testsuite/MDAnalysisTests/plugins/open_files.py 
> <https://github.com/MDAnalysis/mdanalysis/pull/874#discussion_r66750705>:
>
> >
> > -        If several file handle are open for a given file, then the path will
> > -        appear as many time as they are open file handle for that file.
> > -        """
> > -        process = psutil.Process(os.getpid())
> > -        handlers = process.open_files()
> > -        path = [f.path for f in handlers]
> > -        return path
> > +def _gather_open_files(self):
>
> you need to remove the |self| argument.
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub 
> <https://github.com/MDAnalysis/mdanalysis/pull/874/files/5603421ec8ecc5045a844088c8b284ee82c50806..64f0143ff39a691016cdd0f05d25c4a32673e3cc#r66750705>, 
> or mute the thread 
> <https://github.com/notifications/unsubscribe/ABUWuiox6jeDzyb8Kh13mN-dqUJzghHqks5qLQqEgaJpZM4IzFwH>.
>